frontend/src/app/main/data/workspace.cljs

```diff
@@ -355,6 +356,13 @@
                      (rx/take 1)
                      (rx/map zoom-to-frame)))
 
+              (when (:comment-id rparams)
```

Comment:
Is a minor issue but I think we can follow the same stile as prevous code:
![image](https://github.com/user-attachments/assets/b29ee263-ff28-4e18-bc34-2ef8df27f5fb)

Reply:
OK